Documentation/Upgrade/rook-upgrade.md

```diff
@@ -52,7 +52,7 @@ those releases.
 
 ## Breaking changes in v1.16
 
-* The minimum supported Kubernetes version is v1.27.
+* The minimum supported Kubernetes version is v1.28.
```

Comment:
Have you rebased? I believe Blaine already changed this in his upgrade doc PR.

Reply:
done now.